src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c

```diff
@@ -225,15 +225,40 @@ zocl_fpga_mgr_load(struct drm_zocl_dev *zdev, char *data, int size)
 	return err;
 }
 
+static int
+zocl_load_partial(struct drm_zocl_dev *zdev, const char *buffer, int length)
+{
+	int err;
+	void __iomem *map = NULL;
+
+	if (!zdev->pr_isolation_addr) {
+		DRM_ERROR("PR isolation address is not set");
+		return -ENODEV;
+	}
+
+	map = ioremap(zdev->pr_isolation_addr, PR_ISO_SIZE);
+	if (IS_ERR_OR_NULL(map)) {
+		DRM_ERROR("ioremap PR address failed");
```

Comment:
I suggest we log the pr_isolation_addr here

Reply:
added, thanks.